addons/web/static/tests/views/fields/one2many_field_tests.js

```diff
@@ -6730,6 +6730,68 @@ QUnit.module("Fields", (hooks) => {
         await clickSave(target);
     });
 
+    QUnit.test(
+        "one2many list editable - should properly unselect the list field after shift+tab",
+        async function (assert) {
+            await makeView({
+                type: "form",
+                resModel: "partner",
+                serverData,
+                arch: /* xml */ `
+                <form>
+                    <group>
+                        <field name="display_name"/>
+                        <field name="turtles">
+                            <tree editable="bottom">
+                                <field name="turtle_foo"/>
+                                <field name="turtle_bar" optional="hide"/>
+                            </tree>
+                        </field>
+                    </group>
+                </form>`,
+                resId: 1,
+            });
+
+            await click(target, ".o_data_row td:first-child");
+            assert.containsOnce(target, ".o_selected_row", "should have selected row");
+            const { keydownEvent } = triggerHotkey("shift+tab");
```

Comment:
Generally it's better to capitalize each part of the hotkey in triggerHotkey, as this util will split these parts and generate standard keydown/up events.

So better practice to write triggerHotkey("Shift+Tab") here, although it changes nothing for this test.
(I know, there are plenty of usage that don't respect this)